content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
 {{% /capture %}}
 
 {{% capture body %}}
-Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
-declarative configuration and automation. It has a large, rapidly
-growing ecosystem. Kubernetes services, support, and tools are widely available.
+Kubernetes is a cloud infrastructure that provides for the deployment and
```

Comment:
Is Kubernetes an infrastructure?

I think of an infrastructure (in the IT sense) as something that I can run a specific workload on. I _can_ run a workload on a Kubernetes infrastructure, and that might well be in the cloud.

Reply:
K. I will change infrastructure to platform